docs/legacy/sunset.md

```diff
@@ -43,11 +43,19 @@ Det er ingen plan om å avvikle de gamle ingressene i FSS, men vi anbefaler en g
 
 ### MQ
 
+MQ benyttes i dag kun via on-premises servere som driftes og vedlikeholdes av ATOM og Linux. 
 
+Det er ønskelig at de applikasjonene som har mulighet heller benytter Aiven Kafka. Det er dog støtte for MQ fra nais-klusterne on-premises og i GCP, men da kun med nye serverne som er satt opp med autentisering.
```

Comment:
Skulle vi sagt noe om hvorfor vi ønsker at applikasjoner benytter Aiven Kafka i stedet for MQ?

Reply:
> Skulle vi sagt noe om hvorfor vi ønsker at applikasjoner benytter Aiven Kafka i stedet for MQ?

Det er vel et resultat av ønsker fra Teknisk retning, så kanskje @audunstrand har lyst til å prøve seg på en formulering?